Renderer.js

```diff
@@ -0,0 +1,60 @@
+
+class Renderer {
+    constructor(apiManager) {
+        this.data = apiManager
```

Comment:
You should not access the APIManager (`Model`), inside the Renderer (`View`). The connection between them should be in the Main file (`Controller`)